wscript

```diff
@@ -32,14 +35,44 @@ elif sys.platform.startswith('freebsd'):
 elif sys.platform.startswith('linux'):
   # These executable depend on libtinfo.so.5
   CLANG_TARBALL_NAME = 'clang+llvm-$version-linux-x86_64-ubuntu14.04'
+elif sys.platform == 'win32':
+  CLANG_TARBALL_NAME = 'LLVM-$version-win64'
+  CLANG_TARBALL_EXT = '.exe'
 else:
-  # TODO: windows support (it's an exe!)
   sys.stderr.write('ERROR: Unknown platform {0}\n'.format(sys.platform))
   sys.exit(1)
 
 from waflib.Tools.compiler_cxx import cxx_compiler
 cxx_compiler['linux'] = ['clang++', 'g++']
 
+if sys.version_info < (3, 0):
```

Comment:
Is this shim for https://docs.python.org/2/library/os.html#os.symlink (`os.symlink` is only available for Unix-like systems)?

Reply:
Yes. 
In Python 3 `os.symlink` got an additional argument `target_is_directory` and it is required for symbolic linking directories on Windows. To be Python 2 compatible we have to wrap os.symlink even on Unix-like systems. It is (and should be) simply ignored on Unix-like systems as per https://docs.python.org/3/library/os.html#os.symlink

Reply:
I think put these into a separate file is better (e.g. compat.py).